api/src/index.js

```diff
@@ -108,42 +109,19 @@ DiscordService.init().then(() => {
   initCallbacksForQueues().then(() => {
     console.log("Callbacks for queues initialized");
   });
-=======
-DiscordService.init().then(async () => {
-  await queueService.onStartup();
-  await resultRankedService.onStartup();
-});
-
-
-async function shutdown() {
-    console.log('Shutting down server...');
-    await queueService.onShutdown();
-    await resultRankedService.onShutdown();
-}
-
-process.on('SIGINT', async () => {
-  console.log('Received SIGINT, shutting down gracefully...');
-  await shutdown();
-});
-
-process.on('SIGTERM', async () => {
-  console.log('Received SIGTERM, shutting down gracefully...');
-  await shutdown();
->>>>>>> a96bcc1 (remove cron and use setInterval in the QueueService)
```

Comment:
Oh, please make sure to merge these back in :) We should call the onShutdown and onStartup methods